package-scripts.js

```diff
@@ -276,7 +276,7 @@ module.exports = {
       },
       postbuild: {
         script:
-          'buildProduction docs/_site/index.html --outroot docs/_dist --canonicalroot https://mochajs.org/ --browsers ">= 1%, last 1 major version, not dead, Chrome >= 45, Firefox >= 38, Edge >= 12, Explorer >= 10, iOS >= 9, Safari >= 9, Android >= 5, Opera >= 30" --optimizeimages --svgo --inlinehtmlimage 9400 --inlinehtmlscript 0 --asyncscripts && shx cp docs/_headers docs/_dist/_headers && node scripts/netlify-headers.js >> docs/_dist/_headers',
+          'buildProduction docs/_site/index.html --outroot docs/_dist --canonicalroot https://mochajs.org/ --browsers ">= 1%, last 1 major version, not dead, Chrome >= 45, Firefox >= 38, Edge >= 12, Explorer >= 10, iOS >= 9, Safari >= 9, Android >= 5, Opera >= 30" --optimizeimages --svgo --inlinehtmlimage 9400 --inlinehtmlscript 0 --asyncscripts && cp docs/_headers docs/_dist/_headers && node scripts/netlify-headers.js >> docs/_dist/_headers',
```

Comment:
Why'd you add all that crap to the cmdline when it could have been added to "package.json" in `]browserlist](https://github.com/browserslist/browserslist-example#how-to-add-browserslist-to-your-project)`?

Reply:
Turns out assetgraph-builder didn't quite behave as documented and didn't let you use autoprefixer without configuring like this. A patch for that is on the way. Then we can switch to `.browserslit`

Reply:
Why create external file instead of using "package.json"?

Reply:
Because that's the convention we have followed for every other tool that has this choice, and because it's the most common convention I've seen people use, which will increase the chances of them seeing something they can relate to

Reply:
I find this file confusing since my impression would be that it implies what Mocha the runtime supports, not browser is needed to view the website.

Reply:
Was any consideration ever given to migrating the website to its own repository?

Reply:
The website was migrated from its own repository into the main repository in order to version it together with the library.

If you objection to the root location is potential confusion with what browsers mocha.js supports, we can move this file into `/docs/`

Reply:
Moving into "/docs/" would reduce confusion, IMO. Adding a comment to the top of the file indicating the setup relates to website generation would be much appreciated as well.

Trying to read the `[browserslist](https://github.com/browserslist/browserslist)` documentation to see how to support _both_ concepts; if we have settings in "package.json" to imply runtime, would that cause issues with the website build?

Reply:
Use `"BROWSERSLIST_CONFIG"` environment variable for website generation.

Reply:
I fail to see the aversion against having a config file named intuitively after what it does. An environment variable would be declared some non-obvious place in package-scripts, which is already quite noisy.

I moved the config file to `/docs/` and adjusted the browser range to be more modern

Reply:
The browserslist config file, now moved into "/docs/", is fine.

What I was attempting to communicate was that _if_ we added "browserslist" to "package.json" to represent runtime, the environment variable _could be used_ to specify that the "/docs/.browserlistrc" be used for website generation.

Have no experience with the package -- "BROWSERSLIST_CONFIG" may not be necessary, but wanted to ensure that the default lookup order could be overridden -- not sure it would be immediately obvious which was being used otherwise.